code-for-charlottesville/housinghub
# Connect user table finish register

## Proposed Changes

- finish out the register endpoint
- connect db, create users table

## How to test

- follow instructions in the readme to create a sample user and check that it's created in the DB
